### DISCUSSION TITLE
【委托】近期玩具店有几只邦布走失，目前估计在空洞中，在线请求绳匠帮助！

### DISCUSSION THREAD

**Discussion by FANG-12-pixel** [share121/inter-knot#2460] [其他]
![微信图片_20240816225640](https://github.com/user-attachments/assets/e1b30794-4ef3-45ec-ada0-3d3b28181506)
【委托】近期玩具店有几只邦布走失，目前估计在空洞中，在线请求绳匠帮助！
![微信图片_20240816230103](https://github.com/user-attachments/assets/115ea64d-5c23-4f18-96f5-a6c7355db60b)
【注】报酬：菲林*500

**Comment by rikka-612** [comment]
这个委托我接了

**Comment by haily12345** [comment]
简介已发，我来接！

**Comment by mingyu818** [comment]
这个委托我接了！

**Comment by NekoWise** [comment]
第一只邦布真像我们维多利亚家政的巴特勒先生啊。
